templates/index.template.html

```diff
@@ -106,8 +109,8 @@ <h2 class="display-4">Proud of our history</h2>
         <div class="col-md-8 order-1">
           <div class="p-5">
             <h2 class="display-4">Contact Us</h2>
-            <p>If you have questions, suggestions or comments about any of these projects (or if you want to join us) please contact the dev team
-              at <a href="mailto:inakos@googlegroups.com">inakos@googlegroups.com</a></p>
+            <p>If you have questions, suggestions or comments about any of these projects please contact the dev team
```

Comment:
I think the text and anchor were worth having. Why removing them?

Reply:
There was a wrong correction on the template. It should be ok now.